### DISCUSSION TITLE
No active transaction while truncating the temp table

### DISCUSSION THREAD

**Discussion by abdulwahhabkhan** [laravel/framework#38417] [Q&A]
- Laravel Version: 8.54.0
- PHP Version: 8.0.2
- Database Driver & Version:

### Description:

![image](https://user-images.githubusercontent.com/4024244/129720944-5ae56c53-e718-4a6a-9086-4bee8ba17722.png)

### Steps To Reproduce:
Truncate statement within a store procedure, 

I have an SP that pull the data from table store it in temp table do the processing and update the data back to the original table. It truncates the temp table.

**Comment by spector32** [comment]
Still an issue on Laravel 8.83.27
